src/content/app/regulatory-activity-viewer/components/region-overview/region-overview-image/transcription-start-site/TranscriptionStartSite.tsx

```diff
@@ -43,10 +43,43 @@ type Props = {
   scale: ScaleLinear<number, number>;
 };
 
-const TranscriptionStartSite = (props: Props) => {
-  const { yStart, yEnd, tss, strand, scale } = props;
+/**
+  Design guidelines from Andrea:
+
+  shorter arrow
+  height: 12
+  horizontal: 6
+
+  arrowhead:
+  base 6
+  height: 6.75
+
+
+
+  longer arrow
+  height: 19
+
+
+  distance between: 1?
 
-  const stemX = scale(tss[0].position);
+
+  distance from gene: 5
+ */
```

Comment:
Do these comments belong somewhere else?

Reply:
No, they do in fact belong in this component 😄  These are Andrea's instructions for how to update it, but I have not yet got an opportunity to update it. I can take it out if you like; but it will be more convenient to leave it here and to remove it after the component has been updated in a subsequent PR.